### PR TITLE
delete postgres installation

### PR DESCRIPTION
**Pull request by td928**
address issue #325 

# overview
In my attempt to fix the error thrown by prod2, I found the init step was causing an installation issue with postgres. The solution here is to remove the installation step from init. This leads me to find a greater issue with PLUTO github. 

## CI & CI test 
The reason why the issue wasn't spotted earlier was due to at development stage all changes are tested using [test.yml](https://github.com/NYCPlanning/db-pluto/blob/main/.github/workflows/test.yml) and one significant difference between the test and CI is using our self-hosted server vs. running out of github hosted runners with docker postgis image. This is probably we weren't able to catch the CI dependencies issues earlier. 

## Solutions (?)
First one is the one being carried out in this feature branch. Remove the unnecessary installation step to avoid the conflicts and make better documentation of the required setup needs to be done to our build engine (prod2) before pluto production. 

Alternatively, we switched over everything from running on self-hoseted to github hosted streams. Since the most of the test runs [have been successfully executed by the github-hosted runners](https://github.com/NYCPlanning/db-pluto/actions/workflows/test.yml).

 It was noted by @AmandaDoyle that we will lose the intermediate steps in the build with github hosted runner and probably make debugging more challenging. But I found it equally tricky thing trying to set up our build engine (prod2) to perfectly mirror the environment set up by docker which is really the main holdup for pluto build up to this point.

### CODE REVIEW

File: pluto_build/bin/setup.sh [hunk 1]
```diff
@@ -4,17 +4,8 @@ function setup {
...
-# get the signing key and import it
-curl -O https://www.postgresql.org/media/keys/ACCC4CF8.asc
-sudo apt-key add ACCC4CF8.asc
-
-sudo apt update
-sudo apt install -y postgresql-client-11 gdal-bin
-sudo apt autoremove -y
-rm ACCC4CF8.asc
+# install pip for the workflow to run correctly
+sudo apt install python3-pip
```
Comment (SashaWeinstein) [inline pluto_build/bin/setup.sh:8]: I had no idea this setup.sh file existed lol. I can't figure out where the `setup` bash function is run. Is it in the last line with the `register` call?

**Comment by SashaWeinstein** [comment]
I have several questions:
1. What's the difference between the CI workflow and the Publish workflow?
2. What are the "intermediate steps in the build" that we would lose using github hosted runners?
3. Where is the `setup` bash function run? Why is the bash code in a .sh file instead of the workflow yml?

**Comment by td928** [comment]
> I have several questions:
> 
> 1. What's the difference between the CI workflow and the Publish workflow?
> 2. What are the "intermediate steps in the build" that we would lose using github hosted runners?
> 3. Where is the `setup` bash function run? Why is the bash code in a .sh file instead of the workflow yml?

@SashaWeinstein I believe in 1 you are referring to the [CI](https://github.com/NYCPlanning/db-pluto/blob/main/.github/workflows/issue-comment-runner.yml) and [CI-test](https://github.com/NYCPlanning/db-pluto/blob/main/.github/workflows/test.yml) and difference is one use our self-hosted prod2 and one is using github-hosted runner. 

for 2 I think potentially all those intermediate table created in build step (e.g. [this sql table](https://github.com/NYCPlanning/db-pluto/blob/main/pluto_build/sql/_create.sql)) would be queriable in the postgres db lived on prod2 if we used self-hosted.

for 3 see below which are identical in CI-test and CI https://github.com/NYCPlanning/db-pluto/blob/2438d0eba8a73c66d95791d3260b536a8718db9b/.github/workflows/issue-comment-runner.yml#L29-L34

File: pluto_build/bin/setup.sh (hunk 1 above)
Comment (td928) [reply pluto_build/bin/setup.sh:8]: see my answer below for this.

**Comment by SashaWeinstein** [comment]
1. Ok sorry that was a silly question.
2. I'm all about moving away from the postgres DB on our self-hosted runner, don't think it's a good setup in general. If we need to debug along the way we should be running PLUTO on our local machines. We could also work on improving logs 
3. Ok I see. The workflow points to the `pluto` file (no .sh file extension for some reason), which sources from the cli.sh to run the setup.sh. Far too confusing imo, don't think we need this level of complexity

**Comment by td928** [comment]
> So is the idea here to change setup.sh and then run with the action that points to the github runner? So no changes to the workflow yml's needed, just run the one we think is best?
> 
> When I get an answer to that question I'll feel good about approving

yeah that's right I think since we are leaning all toward a github runner build I am all behind it as well. Let me make the changes in this branch

**Comment by td928** [comment]
sorry one more change impending. The CI test does not use recipe engine which we will also need for CI

File: pluto_build/sql/zoning_specialdistrict.sql [hunk 2]
```diff
@@ -3,7 +3,7 @@
 -- the order special purpose districts are assigned is based on which district covers the majority of the lot
 -- a district is only assigned if more than 10% of the district covers the lot
 -- OR more than a specified area of the lot if covered by the district
-DROP TABLE specialpurposeperorder;
+DROP TABLE IF EXISTS specialpurposeperorder;
```
Comment (mbh329) [inline pluto_build/sql/zoning_specialdistrict.sql:6]: Why was `IF EXISTS` added to these two sql files

**Review by mbh329 — APPROVED** [review]

**Pull request merged**